erpnext_oralcare/erpnext_oralcare/doctype/family/family.json

```diff
@@ -0,0 +1,38 @@
+{
+ "creation": "2019-05-15 01:11:55.274550",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "name1"
+ ],
+ "fields": [
+  {
+   "fieldname": "name1",
```

Comment:
name is reserved for id of the doctype so when you use "name" as the id of any field it changes to name1, name2 and so on. Dont use name ever change it to something more meaningful like family_name

Reply:
done